src/components/molecules/SectionRenderer/SectionRenderer.tsx

```diff
@@ -225,7 +225,13 @@ function SectionRenderer(props: SectionRendererProps) {
             options={itemRendererOptions}
             indentation={sectionIndentation || 0}
           />
-        ))}
+        ))
+      ) : (
+        <S.EmptyGroupText>
+          {sectionBlueprint.customization?.emptyVisibleItemsText || 'No items found.'}
```

Comment:
Is it intended that this impacts the resource navigator?

<img width="788" alt="Screenshot 2022-05-19 at 09 17 03" src="https://user-images.githubusercontent.com/7761005/169234060-acef5bf5-a298-4338-bd63-98421df89104.png">